microsoft/testsuites/xdp/common.py

```diff
@@ -151,7 +151,7 @@ def _aggregate_count(
     patterns: List[Pattern[str]],
 ) -> int:
     ethtool = node.tools[Ethtool]
-    nic_names = [nic.upper, nic.lower]
+    nic_names = [nic.name, nic.pci_device_name]
```

Comment:
This is to get all nics' names. Is it included the lower nic name?

Reply:
I didn't change its logic

Reply:
It should be like `nic_names = [nic.name, nic.lower]`. It's easier to understand. My understanding is the `pci_device_name` should be presented in pci nic in normal case, instead of the upper synthetic nic.

Reply:
updated